### PR TITLE
Use specific commit references

### PR DESCRIPTION
**Pull request by orta**
currently I was kinda hoping that it would just work with `HEAD` and `master` - but #34 shows we're not. So I'm taking out the commit references from ENV vars on all our CI sources and using those.

Not writing tests for this till I know it works on a real PR.. like this one.

### CODE REVIEW

**Comment by orta** [comment]
OK, this is where we want to be :+1: bunch of testing changes to do though

**Comment by ashfurrow** [comment]
I've used this on a PR and got [this response](https://github.com/artsy/eigen/pull/1059#issuecomment-174085034):

    &nbsp; | 4 Messages
    ------------- | ------------
        :book: | Insertions: 136
        :book: | Added: ["Artsy/App/ARHockeyFeedbackDelegate.swift"]
        :book: | LOC: 206
        :book: | Mod: ["Artsy.xcodeproj/project.pbxproj", "Artsy/App/ARAppDelegate+Analytics.m", "Artsy/App/ARAppDelegate.m", "Artsy/App/ARHockeyFeedbackDelegate.swift", "Artsy/App/AROptions.h", "Artsy/App/AROptions.m", "Artsy/App/ARSwitchBoard.m", "Artsy/App/ARSwitchboard+Eigen.m", "Artsy/Resources/Artsy-Bridging-Header.h", "Artsy/View_Controllers/Admin/ARAdminSettingsViewController.h", "Artsy/View_Controllers/Admin/ARAdminSettingsViewController.m", "Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m", "Artsy/View_Controllers/Embedded/Generics/AREmbeddedModelsViewController.m", "Artsy_Tests/View_Controller_Tests/App_Menu/ARTopMenuViewControllerSpec.m", "CHANGELOG.yml", "Dangerfile", "Gemfile", "Gemfile.lock"]
    

Hard to tell if it's accurate since I've merged in upstream changes from master, which it appears to be including as well.

**Comment by orta** [comment]
Great, this functions
screen shot 2016-01-25 at 20 14 09

**Comment by orta** [comment]
OK, this is good to go :+1:

File: Dangerfile [hunk 1]
```diff
@@ -20,4 +20,4 @@ end
 
 unless files_modified.any? { |a| a.include?("CHANGELOG.md") }
   warn("Please include a CHANGELOG entry")
-end
+end
```
Comment (KrauseFx) [inline Dangerfile]: No new line

**Comment by KrauseFx** [comment]
Looks good to me, just added one comment

**Comment by orta** [comment]
Cool, added - will merge on green.

**Pull request merged**